src/box/lua/schema.lua

```diff
@@ -403,16 +405,25 @@ local begin_options = {
         return true
     end,
     txn_isolation = normalize_txn_isolation_level,
+    is_sync = function(is_sync)
```

Comment:
Please also add `IS_SYNC` flag to the protocol:
one might start a transaction over the net by issuing a `IPROTO_BEGIN` request. You need to add `IS_SYNC` to `IPROTO_BEGIN` options.

See `xrow_decode_begin()`, `tx_process_begin()` for reference.

Reply:
**There are several problems here.**
1. At the encoding stage, you need to understand whether `is_sync = false` explicitly or not (`is_sync = nil`). I have to do so:
```C
// iproto_constants.h

_(IS_SYNC_BOOL, 0x48, MP_BOOL) \
_(IS_SYNC_NIL, 0x49, MP_NIL)   \
```
to be able to separate `bool` from `nil` for `is_sync` option. It doesn't look very good. How can this be done better without kludge?